packages/core/test/actions.test.ts

```diff
@@ -1474,7 +1485,7 @@ describe('initial actions', () => {
     }
   });
 
-  it.skip('should support initial actions', () => {
+  it('should support initial actions', () => {
     // TODO: fix initial state actions on root node
```

Comment:
is this comment still relevant? I don't see anything in the `machine` that would be missing from this expectation
```suggestion
```